azure-pipelines.yml

```diff
@@ -28,31 +28,25 @@ jobs:
     parameters:
       jobName: Linux_Java_8
       timeoutInMinutes: 180
-      pool: tc_linux_pool
-      # options: -B -Djava.test.version=1.8 -Djava.test.vendor=zulu -Dfast -DfailIfNoTests=false -DskipITs -Dtest=UniversalPathTest
-      options: -B -Djava.test.version=1.8 -Djava.test.vendor=zulu
+      options: -B -Djava.test.version=1.8
       mavenGoals: 'clean install'
   - template: build-templates/maven-common.yml@templates
     parameters:
       jobName: Linux_Java_11
       timeoutInMinutes: 180
-      pool: tc_linux_pool
-      # options: -B -Djava.test.version=1.8 -Djava.test.vendor=zulu -Dfast -DfailIfNoTests=false -DskipITs -Dtest=UniversalPathTest
-      options: -B -Djava.test.version=1.8 -Djava.test.vendor=zulu
+      options: -B -Djava.test.version=1.11
```

Comment:
It looks like the 1.11 jobs had been mistakenly moved to 1.8 at some point... I've fixed that here as a side-effect.